pkg/apis/psp/types.go

```diff
@@ -0,0 +1,283 @@
+/*
```

Comment:
will you delete the old internal types for PSPs?

Reply:
I'm following a roadmap that @soltysh gave me: in this Kubernetes release, I only need to introduce a new API that will be stored as old PSP types. So, in the current iteration, I won't remove anything from PSP.

Reply:
Which doubles the code for the internal types. You can easily share them by removing the old extensions types and let the extensions group point to your new ones in the new group. Compare https://github.com/kubernetes/kubernetes/pull/50327.

Reply:
Maybe I'm missing something here, but I don't see how it doubles the code for the internal types. New API group is using the old internal types.